src/website/apps/api/v2/viewsets/publications.py

```
@@ -32,12 +36,17 @@
+    # Slug configuration
+    slug_filter_fields = ['slug']  # Publication uses standard slug field
+
     # Optimization settings
     select_related_fields = []  # No foreign keys to optimize
     prefetch_related_fields = []  # No M2M relationships
     list_only_fields = [
+        # Model fields for list view optimization
         'id', 'title', 'authors', 'category', 'link', 'link_title',
         'resource_file', 'order', 'created', 'modified'
+        # Note: Slug fields added automatically by mixin
     ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)